week1/645_set_mismatch.py

```diff
@@ -0,0 +1,11 @@
+# https://leetcode.com/problems/set-mismatch/
+def set_mismatch(nums: list[int]) -> list[int]:
+    freq_dict = {i + 1: 0 for i in range(len(nums))}
+    for n in nums:
+        freq_dict[n] += 1
+
+    freq_list = list(freq_dict.values())
+    if 0 not in freq_list:
```

Comment:
This will never happen. You can carefully read the description and figure out that it will ALWAYS be the case that one element will be missing and one number will have a duplicate so this condition is never met. This is unnecessary error checking.